manifests/install.pp

```diff
@@ -1,10 +1,10 @@
 class deploynaut::install inherits deploynaut {
 
   package { 'redis-server':
-    ensure => present
+    ensure => '2:2.8.17-1+deb8u5'
```

Comment:
this is kind of pointless, and means we're stuck at this version with no security updates, the actual version doesn't change in point releases, only on major debian versions, so I would suggest not doing this.